src/asciidoc/docs/platform-features/entitlements.adoc

```diff
@@ -12,39 +12,37 @@ When you initialize the @withkoji/core package with this configuration data, the
 
 The following platform entitlements are available for Koji apps.
 
-== Contexts
+== Configuration and navigation
+
+To allow creators to configure your app, you must enable the associated entitlement.
+Your app can also enable or disable some platform navigation features.
 
-Your app can provide different experiences for different modes, called *contexts*.
-To enable certain contexts, you must enable the associated entitlement.
-To determine the current context of a running app, use <<core-frontend-playerstate#context, Koji.playerState.context>>.
 
 [source,json]
 ----
 {
   "entitlements": {
     "InstantRemixing": true, <1>
-    "AboutContext": true, <2>
-    "AdminContext": true, <3>
-    "StickerContext": true <4>
+    "InstantRemixingNativeNavigation": true, <2>
+    "InstantRemixingNativeCancel": true, <3>
+    "FeedEvents": true <4>
   }
 }
 ----
 <1> `InstantRemixing` – _Boolean_, Allows creators to configure the app (`remix` context).
 Default is `false`.
-Optionally, configure entitlements to <<_navigation, control the navigation>> for the configuration experience.
-<2> `AboutContext` – _Boolean_, Enables a distinct experience when a creator first configures the app (`about` context).
+To determine the current context of a running app, use <<core-frontend-playerstate#context, Koji.playerState.context>>.
+<2> `InstantRemixingNativeNavigation` – _Boolean_, Hides the platform action for advancing from a configuration to a preview, so that the app can provide this action.
 Default is `false`.
 +
-NOTE: If this entitlement is enabled, the app must use <<core-frontend-ui-navigate#createRemix, Koji.ui.navigate.createRemix>> to open the configuration options from the About page.
+NOTE: If this entitlement is enabled, the app must use <<core-frontend-remix#finish, Koji.remix.finish>> to advance to the preview.
 
-<3> `AdminContext` – _Boolean_, Enables a distinct experience for the creator (`admin` context).
-For example, fulfillment for product sales apps or content moderation for social post apps.
-If enabled, the creator can access the admin page by going to *App > ... > Admin Page*.
-Default is `false`.
-<4> `StickerContext` – _Boolean_, Enables a distinct experience when the app is embedded in another Koji app (`sticker` context).
+<3> `InstantRemixingNativeCancel` – _Boolean_, Hides the platform action for cancelling a configuration, so that the app can provide this action.
 Default is `false`.
++
+NOTE: If this entitlement is enabled, the app must use <<core-frontend-remix#cancel, Koji.remix.cancel>> to implement a custom cancel action.
 
-NOTE: To use the `receipt` context, you must <<_in_app_purchases, configure in-app purchases>> and enable dynamic receipts.
+<4> `FeedEvents` – _Boolean_, Enables the app to be displayed in a platform feed.
```

Comment:
This one is also deprecated! Sorry for not catching on the first pass 😅

Reply:
Thanks! Removed this one throughout, as well.